app/views/upload.scala.html

```diff
@@ -32,7 +32,7 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
                 <form id="file-upload-form">
                 <div class="upload-form">
                     <label class="govuk-label" for="file-upload">
-                        Select a folder
+                        Select a folder or zip file
```

Comment:
This came up in the meeting earlier, but we'll have to think about how to present this because you can't have a Browse button that will allow you to select a folder _or_ a zip file. Either the site will have to choose the input type depending on the browser, or we'll have to ask the user to choose.

Reply:
Absolutely right. Thank you Suzanne!